web/src/components/task/task-modal/task-validation-modal.tsx

```diff
@@ -45,16 +44,17 @@ export const FinishTaskValidationModal: FC<IProps> = (modalProps) => {
         allUsers,
         invalidPages,
         editedPageCount,
-        isOwner,
         allValid,
+        isOwner,
         abort,
         validSave,
         success,
         onRedirectAfterFinish,
         onSaveForm
     } = modalProps;
+
     const [validatorUserId, onValidatorUserIdChange] = useState(
-        allUsers.validators[0]?.id || allUsers.annotators[0]?.id
+        allUsers.validators[0] || allUsers.annotators[0]?.id
```

Comment:
Please help me understand why you are leaving out the id in this part of the code? changed the field object `allUsers.validators[0]`?

Reply:
now we have validators from job, field "job.validators" contains string[]